blackbox/docs/admin/system-information.rst

```diff
@@ -1441,37 +1441,37 @@ have not been recreated, CrateDB will refuse to start.
 To recreate a table, you have to create new tables, copy over the data and
 rename or remove the old table.
 
-1) Use :ref:`ref-show-create-table` to get the schema required to create an
-empty copy of the table to recreate::
+1. Use :ref:`ref-show-create-table` to get the schema required to create an
+   empty copy of the table to recreate::
 
-    SHOW CREATE TABLE your_table;
+    cr> SHOW CREATE TABLE your_table;
 
-2) Create a new temporary table, using the schema returned from
-:ref:`ref-show-create-table`::
+2. Create a new temporary table, using the schema returned from
+   :ref:`ref-show-create-table`::
 
-    CREATE TABLE tmp_your_table (...);
+    cr> CREATE TABLE tmp_your_table (...);
```

Comment:
Code blocks starting with `cr>` are executed and the output is verified. These now all fail.

Reply:
there is a way to turn this off for specific blocks. I recommend you figure out how to do that instead of dropping ``cr>``

I strongly prefer that we keep our terminal prompts consistent across examples. whether or not a code example is part of our test suite should be transparent to the end-user

Reply:
okay damn it looks like I'm mistaken

here's where the file is added for parsing for doctests:

https://github.com/crate/crate/blob/master/blackbox/test_docs.py#L459

and here's the `cr>` parser:

https://github.com/crate/crate/blob/master/blackbox/test_docs.py#L131

ideally, there would be a way to specify to the parser that some blocks can be skipped. but I don't know enough about our testing setup to know how feasible this is. what are your thoughts, @MarkusH?

I think I misremembered, and in the past, I've just been creative with which files I put which examples in

another solution here is to make sure that your examples do run and execute without errors. you can do this with hidden code blocks (e.g., https://github.com/crate/crate/blame/master/blackbox/docs/admin/system-information.rst#L42)